### ISSUE TITLE
md-content scroll ignores flex

### ISSUE THREAD

**Issue by derekmoss** [angular/material#5340]
When setting up a view with partial scrolls in the same column, 
it seems no matter how I slice it, md-content will ignore the heights set by flex
if its scroll extent is larger than another md-content's scroll extent in the same column
the height gets larger for the md-content with the larger scroll extent.

interactive height example
http://codepen.io/jayray/pen/yYvwMZ

Ideally if had a column and had it divided in two,
the heights should always be 50% 50%

whether thats just 
```html
<md-content layout="column">
    <md-content flex></md-content>
    <md-content flex></md-content>
</md-content>

<md-content layout="column">
    <md-content flex="50"></md-content>
    <md-content flex="50"></md-content>
</md-content>
```

**Comment by derekmoss** [comment]
This may be related to #4022
Can someone verify if these are the same?

**Comment by topherfangio** [comment]
@derekmoss We don't really support nested `md-content`s. Is there a reason you are using the `md-content` instead of a plain div?

You may need to add the `layout-fill` directive as I have here, but this seems to fix your problem: http://codepen.io/topherfangio/pen/dYmaog?editors=100

**Comment by derekmoss** [comment]
After 11.0 came out, <code>div</code> wouldn't work in firefox or internet explorer.
<code>md-content</code> was the only thing that would support all 4 major browsers.
at least in an easy to use fashion.

basically nest <code>md-content</code> till you get to the piece that scrolls, everything after that needed to be a <code>div</code>.

I'm a glad that this works in Chrome at least, let me check the other browsers.

**Comment by derekmoss** [comment]
http://codepen.io/topherfangio/pen/dYmaog?editors=100 doesn't work in firefox

**Comment by topherfangio** [comment]
@derekmoss Indeed, I believe this is a bug with Firefox not properly calculating the height of flex elements.

I updated my example to show the two solutions:

1. Use `flex="50"` as you mentioned, or
2. Set the overflow to something (I did `overflow: auto`)

Both of these force Firefox to properly calculate the height. We're talking about the best ways to support/document this as there are cases where you just want it to flex and fill the space, but if you don't set the overflow, it looks wrong in Firefox.

Unfortunately, globally changing the overflow on all .flex items causes lots of other problems, so it's obviously not ideal :-)
